src/screens/rewards/components/RewardsTotalEarnings.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Image } from 'react-native';
+import { RewardsSectionCard } from '@/screens/rewards/components/RewardsSectionCard';
+import {
+  Bleed,
+  Box,
+  Columns,
+  Inline,
+  Rows,
+  Stack,
+  Text,
+} from '@/design-system';
+import { useTheme } from '@/theme';
+import { RewardsProgressBar } from '@/screens/rewards/components/RewardsProgressBar';
+import * as i18n from '@/languages';
+
+const TOTAL_SUPPLY = 2000;
+
+type Props = {
+  totalEarningsUsd: number;
+  totalEarningsToken: number;
+  multiplier: number;
+  tokenSymbol: string;
+  tokenImageUrl: string;
+};
+
+export const RewardsTotalEarnings: React.FC<Props> = ({
+  totalEarningsUsd,
+  multiplier,
+  tokenImageUrl,
+  totalEarningsToken,
+  tokenSymbol,
+}) => {
+  const { colors } = useTheme();
+
+  return (
+    <RewardsSectionCard>
+      <Rows space="16px">
+        <Columns>
+          <Stack space="12px" alignHorizontal="left">
+            <Text color="labelTertiary" size="15pt" weight="semibold">
+              {i18n.t(i18n.l.rewards.total_earnings)}
+            </Text>
+            <Text color="label" size="26pt" weight="heavy">
+              {`$${totalEarningsUsd}`}
```

Comment:
should we be concerned with the user's default currency settings?

Reply:
I was thinking about this, and I think we can't make this happen because we don't have any currency conversion API integrated, the price comes in in USD from the backend so I guess we should leave it this way

Reply:
I could ask them about it though

Reply:
would be nice to have this in something that allows this but fine w me for now

Reply:
I will make it happen later on, we have an idea how to do it without that being returned from the API

Reply:
@benisgold https://linear.app/rainbow/issue/APP-383/use-users-preferred-currency-leveraging-current-op-price-data